docs/src/references/bridged-topics.md

```diff
@@ -63,3 +63,19 @@ The Azure topics are prefixed by `az/`.
  * `az/messages/devicebound/#` - Use this topic to subscribe for the messages that were sent from cloud to device.
  Any message published by Azure on one the subtopics of `devices/{device_id}/messages/devicebound/#`
  is republished here.
+ 
+ 
+## Collectd topics
+
+When the [device monitoring feature is enable](../tutorials/device-monitoring.md),
+monitor metrics are emitted by `collectd` on a hierarchy of MQTT topics.
+
+* `collectd/$HOSTNAME/#` - All the metrics collected on the device (which hostname is `$HOSTNAME`).
+* `collectd/$HOSTNAME/$PLUGIN/#` - All the metrics collected by a given collectd plugin, named `$PLUGIN`.
+* `collectd/$HOSTNAME/$PLUGIN/$METRIC` - The topic for a given metric, named `METRIC`.
```

Comment:
```suggestion
* `collectd/$HOSTNAME/$PLUGIN/$METRIC` - The topic for a given metric, named `$METRIC`.
```
just to keep it consistent with the other lines

Reply:
fixed